src/Orleans/Runtime/OutsideRuntimeClient.cs

```diff
@@ -904,5 +904,10 @@ public IGrainMethodInvoker GetInvoker(int interfaceId, string genericGrainType =
         {
             throw new NotImplementedException();
         }
+
+        public void BreakOutstandingMessagesToDeadSilo(SiloAddress deadSilo)
+        {
+            throw new NotImplementedException();
```

Comment:
@ReubenBond , this PR only does the silo side. The client side will be implemented separately. But the `IRuntimeClient` interface is the same for cleint ands silo.